source/material_model/rheology/visco_plastic.cc

```diff
@@ -512,6 +519,13 @@ namespace aspect
                            "dynamic stresses are much higher than the lithostatic pressure. "
                            "If false, the minimum pressure in the plasticity formulation will "
                            "be set to zero.");
+        prm.declare_entry ("Use adiabatic pressure in creep viscosity", "false",
```

Comment:
Please update the parameter description based on my comments in the header file.